<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WiFi_Test</class>
 <widget class="QMainWindow" name="WiFi_Test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WiFi_Test</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="wifi_enable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>WIFI Enable</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ssid_edit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>SSID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>WPA2/PWD</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="passwd_edit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Secure Mode</string>
    </property>
    <widget class="QRadioButton" name="secure_none">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="secure_wpa2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>WPA2</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="wifi_stop">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>WIFI Stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>404</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
